apiserver/crossmodel/common.go

```diff
@@ -15,49 +15,17 @@ import (
 	"github.com/juju/juju/apiserver/facade"
 	"github.com/juju/juju/apiserver/params"
 	jujucrossmodel "github.com/juju/juju/core/crossmodel"
-	"github.com/juju/juju/feature"
 	"github.com/juju/juju/permission"
 )
 
-func init() {
-	common.RegisterStandardFacadeForFeature("CrossModelRelations", 1, NewAPI, feature.CrossModelRelations)
-}
-
-// API implements the cross model interface and is the concrete
-// implementation of the api end point.
-type API struct {
+type baseAPI struct {
```

Comment:
I'd rather you created separate packages for the two facades, for consistency. If there's common code, it can go in apiserver/common/crossmodel.